src/sentry/api/serializers/models/group.py

```diff
@@ -568,21 +568,20 @@ def _get_seen_stats(self, item_list, user):
             end=self.end,
         )
 
-        first_seen = {}
         last_seen = {}
         times_seen = {}
+
+        seen_data = tagstore.get_group_seen_values_for_environments(
```

Comment:
Note that this works fine with falsey environments, just doesn't include them as a filter.